pkg/tsdb/influxdb/influxql/converter/converter.go

```diff
@@ -0,0 +1,329 @@
+package converter
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	jsoniter "github.com/json-iterator/go"
```

Comment:
I think we should wrap whatever we need from this import into `jsonitere` so this import is used. That way there won't be a mistake of calling jsoniter (and skipping err lint) when working on the code.

Reply:
Still think this is worth doing

Reply:
I use this to create the initial `jsonitere` instance on line 22
```
iter := jsonitere.NewIterator(jIter)
```
So I think it is ok to stay like this. WDYT?